loops.js

```diff
@@ -33,7 +44,14 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (let i = 0; i < 16 ; i++) {
+    if ( i % 2 == 0){
+        console.log(i + "even")}
 
```

Comment:
Let op de netheid van de code met dubbelepuntkomma:
 console.log(i + ' is even');
  }

Reply:
klopt helemaal ik heb het er nog niet helemaal inzitten en probeer er heel erg op te letten!